package.json

```diff
@@ -33,6 +33,7 @@
     "build:watch": "concurrently \"yarn buildBabel:esm -- --watch\" \"yarn buildTsc -- --watch\"",
     "test": "echo \"Error: no test specified\" && exit 1",
     "storybook": "start-storybook -p 6006",
```

Comment:
Unused?

Reply:
are you refering to `prestorybook`?

I added that because now to run the example the build is needed first.